mne/report.py

```diff
@@ -136,6 +137,9 @@ def _is_bad_fname(fname):
     """Auxiliary function for identifying bad file naming patterns
        and highlighting them in red in the TOC.
     """
+    if fname.endswith('(whitened)'):
+        fname = fname[:-11]
```

Comment:
why is this actually called fname, it's the HTML variables, right, no files involved?